include/trackcpp/passmethods.hpp

```diff
@@ -166,35 +170,34 @@ void bndthinkick(Pos<T>& pos, const double& length,
                  const std::vector<double>& polynom_b,
                  const double& irho,
                  const Accelerator& accelerator,
-                 const double d_factor = 0) {
+                 const double rad_const = 0,
+                 const double qexcit_const = 0) {
 
   T real_sum, imag_sum;
   calcpolykick<T>(pos, polynom_a, polynom_b, real_sum, imag_sum);
   T de = pos.de;
 
-  if (accelerator.radiation_on) {
-    T pnorm = 1 / (1 + pos.de);
+  if (rad_const != 0) {
+    T&& pnorm = 1 / (1 + pos.de);
```

Comment:
should we not redefine pnorm = (1 + pos.de) here too?
I am a bit confused: the two code sections seem inconsistent to me.

Reply:
@xresende, the idea of defining the second appearance of `pnorm` that way was to avoid the unnecessary floating point division operation and substitute the other division by multiplication.

Reply:
the optimization idea I understand. but in one case the variable `pnorm` is set to `(1 + pos.de)` and in another to `1 / (1 + pos.de)`. There seems to be a name inconsistency. maybe we should use `pnorm_inv`in one of the cases.

Reply:
I didn't want to create a new variable. The inconsistency in commented in front of the definition for clarity.